### DISCUSSION TITLE
<pi>ModuleNotFoundError: No module named 'pycaret.nlp'

### DISCUSSION THREAD

**Discussion by dervishali** [pycaret/pycaret#3526] [General]
Dear Colleagues, 

I am following Moez Ali's article I have found on the web : 

https://towardsdatascience.com/topic-modeling-in-power-bi-using-pycaret-54422b4e36d6

I succeeded almost all the steps until I came to Python scripting step : 

from pycaret.nlp import *
dataset = get_topics(dataset, text='en')

When I run the above Python script I get the below error: 
---
DataSource.Error: ADO.NET: Python script error.
<pi>ModuleNotFoundError: No module named 'pycaret.nlp'
</pi>
Details:
    DataSourceKind=Python
    DataSourcePath=Python
    Message=Python script error.
<pi>ModuleNotFoundError: No module named 'pycaret.nlp'
</pi>
    ErrorCode=-2147467259
    ExceptionType=Microsoft.PowerBI.Scripting.Python.Exceptions.PythonScriptRuntimeException
---
And I have found nothing about this error I am getting on the net. 

Please help me how I can get rid of this error to complete all the steps. 

Any help or support will be appreciated. 

Regards, 

Ali

**Comment by ngupta23** [comment]
NLP was deprecated in pycaret 3.0.0. Please use any prior version such as pycaret 2.3.x if you want to use NLP module.

**Reply by dervishali** [reply]
Thank you so much for your prompt reply. I have solved the problem with Pycaret 2.3.10.
